documentation/course/e2/README.md

```diff
@@ -2164,7 +2166,37 @@ TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
 ## Płatność
 
-TODO @piterek130: Dodać diagram pakietów, opis architektury.
+Do implementacji serwisu Inferius wybrano język **Java** z frameworkiem **Spring Boot**.
+
+Wybór ten podyktowany był następującymi czynnikami:
+
+- **doświadczenie zespołu** - większość członków zespołu posiada doświadczenie w programowaniu w języku Java, co pozwoli na spełnienie wymogu wymienionego wyżej - znajomość języka przez co najmniej jedną osobę spoza serwisu w celu przeprowadzenia code review,
+- **dostępność AWS SDK** - AWS SDK wspiera język Java, co pozwoli na łatwe korzystanie z usług AWS,
+- **dostępność bibliotek** - Java posiada bardzo rozbudowaną bazę bibliotek, co pozwoli na szybkie i efektywne rozwijanie serwisu,
+- **dojrzałość frameworka** - Spring Boot jest jednym z najpopularniejszych frameworków do tworzenia aplikacji w języku Java, co pozwoli na szybkie rozwiązywanie ewentualnych problemów,
+- **łatwość w testowaniu** - Java jest językiem, który posiada wiele rozbudowanych narzędzi do testowania, co ułatwi proces testowania serwisu.
```

Comment:
Inconsistent lists: here "- **blah blah** - about blah blah", further down you have "- **blah blah**: about blah blah". Also points should not be bolded, but monospaced instead. Look at Tomek's description.